osu.Game.Tests/Visual/Multiplayer/TestSceneMultiSpectatorScreen.cs

```diff
@@ -314,6 +316,25 @@ public void TestSpectatingDuringGameplayWithLateFrames()
             AddUntilStep("player 2 playing from correct point in time", () => getPlayer(PLAYER_2_ID).ChildrenOfType<DrawableRuleset>().Single().FrameStableClock.CurrentTime > 30000);
         }
 
+        [Test]
+        public void TestPlayersLeaveWhileSpectating()
+        {
+            start(Enumerable.Range(PLAYER_1_ID, 8).ToArray());
```

Comment:
Let's split this enumerable thing into its own function since it's used many times in this scene. Something like

`getPlayerIds(8)`